### PR TITLE
[WIP] Support One Deploy API for Intellij.

### PR DESCRIPTION
**Pull request by shenqianjin**

### CODE REVIEW

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/WebAppWrapper.java [hunk 1]
```diff
@@ -132,4 +134,44 @@ private WebApp getWebApp() {
...
+        this.getWebApp().deploy(type, file, deployOptions);
+    }
+
+    @Override
+    public Completable deployAsync(DeployType type, File file, DeployOptions deployOptions) {
+        return this.getWebApp().deployAsync(type, file, deployOptions);
+    }
+
+    @Override
+    public void deploy(DeployType type, InputStream file) {
```
Comment (andxu) [inline Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/WebAppWrapper.java:159]: bad parameter file for type InputStream

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/WebAppWrapper.java [hunk 2]
```diff
@@ -132,4 +134,44 @@ private WebApp getWebApp() {
...
         }
     }
+
+    @Override
+    public void deploy(DeployType type, File file) {
+        this.getWebApp().deploy(type, file);
+    }
+
+    @Override
+    public Completable deployAsync(DeployType type, File file) {
```
Comment (andxu) [inline Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/WebAppWrapper.java:144]: where is the call to `deployAsync`?

Comment (shenqianjin) [reply Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/WebAppWrapper.java:144]: nowhere. the existing WebAppWrapper implements com.microsoft.azure.management.appservice.WebApp. SDK changed to add this method in its interface on one deploy version. This is forced to implement.

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/WebAppWrapper.java (hunk 1 above)
Comment (shenqianjin) [reply Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/WebAppWrapper.java:159]: the design is owned by java sdk in its interface.

**Pull request closed without merging**